core/src/main/java/org/jclouds/proxy/ProxyForURI.java

```diff
@@ -65,15 +65,12 @@
    public Proxy apply(URI endpoint) {
       if (!useProxyForSockets && "socket".equals(endpoint.getScheme())) {
          return Proxy.NO_PROXY;
-      } else if (config.useSystem()) {
-         System.setProperty("java.net.useSystemProxies", "true");
-         Iterable<Proxy> proxies = ProxySelector.getDefault().select(endpoint);
-         return getLast(proxies);
-      } else if (config.getProxy().isPresent()) {
+      }
+      if (config.getProxy().isPresent()) {
          SocketAddress addr = new InetSocketAddress(config.getProxy().get().getHostText(), config.getProxy().get()
-               .getPort());
+            .getPort());
          Proxy proxy = new Proxy(config.getType(), addr);
-
+         
```

Comment:
should we go to the mailing list to get consensus on whether blank lines should have whitespace to match the prefix on the previous line or the following line or to remove it altogether?  ;)